### PR TITLE
chapter 14　: Following user

### PR DESCRIPTION
**Pull request by daniadore**
Ready

### CODE REVIEW

**Comment by daniadore** [comment]
Ready

File: app/assets/stylesheets/relationships.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Relationships controller here.
```
Comment (anlvq-1058) [inline app/assets/stylesheets/relationships.scss:1]: delete this file please

File: app/helpers/relationships_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module RelationshipsHelper
```
Comment (anlvq-1058) [inline app/helpers/relationships_helper.rb:1]: delete this file please

File: test/models/relationship_test.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
```
Comment (anlvq-1058) [inline test/models/relationship_test.rb:1]: delete this file please

File: test/fixtures/relationships.yml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
```
Comment (anlvq-1058) [inline test/fixtures/relationships.yml:3]: delete this file please

File: test/controllers/relationships_controller_test.rb [hunk 5]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class RelationshipsControllerTest < ActionDispatch::IntegrationTest
```
Comment (anlvq-1058) [inline test/controllers/relationships_controller_test.rb:3]: delete this file please

**Review by ghost — COMMENTED** [review]
cmt

**Comment by daniadore** [comment]
Ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**